decompiler/util/data_decompile.cpp

```diff
@@ -456,6 +456,14 @@ goos::Object decomp_ref_to_inline_array_guess_size(
   // we expect that to be a label:
   ASSERT((field_location % 4) == 0);
   auto pointer_to_data = words.at(field_location / 4);
+
+  // inline-arrays can also be initialized as #f
+  if (pointer_to_data.kind() == LinkedWord::SYM_PTR) {
+    ASSERT_MSG(pointer_to_data.symbol_name() == "#f",
+               "inline-array data decompilation found a non-#f symbol");
```

Comment:
this won't be a great error (though I can't complain too much, I'm sure you guys are tired of dealing with my crappy errors)